low-level/sensors/impl/generic/node_hw.py

```diff
@@ -958,19 +959,30 @@ def _parse_fan_info(self, index, date, _time, device_id, sensor_id,\
         original_event = event
         original_status = status
 
+        sensor_name = self.sensor_id_map[self.TYPE_FAN][sensor_id]
+        impact = "NA"
+        recommendation = None
+        description = ""
+        state = "high" if "going high" in event.lower() else "low"
         if status.lower() == "deasserted" and event.lower() == "fully redundant":
             alert_type = "fault"
+            description = f"{sensor_name} sensor is completely redundant"
         elif status.lower() == "asserted" and event.lower() == "fully redundant":
             alert_type = "fault_resolved"
+            description = f"{sensor_name} sensor has recovered from 'fault' state."
         elif "going high" in event.lower() or "going low" in event.lower():
             if status.lower() == "deasserted":
                 alert_type = "fault_resolved"
+                description = f"{sensor_name} sensor is going {state}"
```

Comment:
`System Fan 1A sensor is going high`,  doesn't really tell me what happened to the Fan @gargpalak048.
Also for the Ipmitool events. assert = fault and de-assert = fault_resolved doesn't match exactly for every event,
```
Sensor States:
  lnr : Lower Non-Recoverable
  lcr : Lower Critical
  lnc : Lower Non-Critical
  unc : Upper Non-Critical
  ucr : Upper Critical
  unr : Upper Non-Recoverable
```
I haven't read in-depth, but these states must be representing some threshold values and event is about going low and high wrt those threshold values. 
so going low wrt upper critical value  - Asserted might be a fault_resolved. (I don't know 🤷‍♂️. This might be totally wrong as well. 😅, but we need to check.)

Reply:
You can use `Lower Non-Recoverable threshold breached for {sensor}` if event is lnr, same for other event

Reply:
@sumedhak27 so in the specific_info, we are parsing the sensor readings and event occurred.
For eg :-  "event": "Asserted - Lower Non-recoverable going low"

earlier the description was same as the event field. 
so to make it more readable, I simply it

> "description": "System Fan 1A sensor is going low",
"impact": "System Fan 1A is not in good health", 
"recommendation": "Please Contact Seagate Support."

what could be more efficient description? Any suggestions?
description: System Fan 1A sensor is in {} state. ( Lower Non-Recoverable, Lower Critical etc) something like this?

@mariyappanp  @sandeepsgt @cdeshmukh any suggestions